package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@metamask/metamask-design-system",
-  "version": "9.0.0",
+  "version": "10.0.0",
```

Comment:
Bumping monorepo version with create release branch

Reply:
why are we bumping a major version?